bundle/Handler/FieldType/CanonicalUrl.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\OpenGraphBundle\Handler\FieldType;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Field;
+use Netgen\Bundle\OpenGraphBundle\MetaTag\Item;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+class CanonicalUrl extends Handler
```

Comment:
Needs to be final

Reply:
Also does not need to extend `Handler`, nor implement `supports` method, since this is not a field type handler. `implements HandlerInterface` is enough.